src/ape_plugins/_cli.py

```diff
@@ -22,38 +22,89 @@ def cli():
     """
 
 
-@cli.command(name="list", short_help="List installed plugins")
+def _remove_prefix_add_version(set_convert, _prefix, version):
+    output = set()
+    text = None
+    if set_convert:
+        for i in set_convert:
+            if i.startswith(_prefix):
+                text = i[len(_prefix) :]
+            if version:
+                output.add(f"{text}    ({version})")
+            else:
+                output.add(text)
+        return output
+    else:
+        return None
+
+
+@cli.command(name="list", short_help="Display all plugins installed and available (including Core)")
 @click.option(
     "-a",
     "--all",
     "display_all",
     default=False,
     is_flag=True,
-    help="Display all plugins (including Core)",
+    help="Display all plugins installed and available (including Core)",
 )
 @ape_cli_context()
 def _list(cli_ctx, display_all):
-    plugins = []
+    installed_second = set()
```

Comment:
`installed_first` second, and so on does not really describe what this is...

`installed_first_class_plugins` while verbose clearly describes it

Reply:
Thank you